frontend/app/src/components/dashboard/EditSnapshotTotal.vue

```diff
@@ -207,9 +213,26 @@ const save = () => {
   input(newValue);
 };
 
-const totalRules = computed(() => [
-  (v: string) => !!v || tc('dashboard.snapshot.edit.dialog.total.rules.total')
-]);
+const rules = {
+  total: {
+    required: helpers.withMessage(
+      tc('dashboard.snapshot.edit.dialog.total.rules.total'),
+      required
+    )
+  }
+};
+
+const v$ = useVuelidate(
+  rules,
+  {
+    total
+  },
+  { $autoDirty: true }
+);
+
+watch(v$, ({ $invalid }) => {
+  set(valid, !$invalid);
+});
```

Comment:
if we only use valid in v-form's value it might make sense to get rid of it and use `v$.$invalid` instead there, the same for other places where this happens.